cpp/include/cudf/io/parquet.hpp

```diff
@@ -555,7 +555,7 @@ class parquet_writer_options {
   // Maximum size of min or max values in column index
   int32_t _column_index_truncate_length = default_column_index_truncate_length;
   // When to use dictionary encoding for data
-  dictionary_policy _dictionary_policy = dictionary_policy::ALWAYS;
```

Comment:
would this change the default for all  codecs or just ZSTD?

Reply:
oh I think this just opens it so the codec check is actually performed.

Reply:
this would apply to all compression codecs

Reply:
All codecs for now. This is just an experimental branch to test `dict_policy` out.